db/data.py

```diff
@@ -124,7 +125,11 @@ def submit_low_level_data(mbid, data):
         )
 
     # The data looks good, lets see about saving it
-    write_low_level(mbid, data)
+    try:
+        write_low_level(mbid, data)
+    except sqlalchemy.exc.DataError as e:
```

Comment:
This doesn't seem like the right place to do that. I'd rather catch exceptions inside `write_low_level`, that's where queries are performed.